core/core.ex

```diff
@@ -48,7 +51,21 @@ defmodule AntikytheraCore do
     System.put_env("ERL_LIBS", new_value)
   end
 
-  defp establish_connections_to_other_nodes(tries_remaining \\ 3) do
+  @connection_retrial_interval_in_milliseconds 5_000
+
+  defunpt calculate_connection_trial_count_from_health_check_grace_period() :: pos_integer do
+    connection_retrial_interval_in_seconds = @connection_retrial_interval_in_milliseconds / 1000
+
+    result =
+      trunc(
+        ClusterConfiguration.health_check_grace_period_in_seconds() /
+          connection_retrial_interval_in_seconds
+      )
+
+    max(result, 1)
```

Comment:
(optional)
```suggestion
    trunc(
      ClusterConfiguration.health_check_grace_period_in_seconds() /
        connection_retrial_interval_in_seconds
    )
    |> max(1)
```

Reply:
or
```suggestion
    (ClusterConfiguration.health_check_grace_period_in_seconds() /
       connection_retrial_interval_in_seconds)
    |> trunc()
    |> max(1)
```

Reply:
Thanks! Let me use the latter.
Fixed in https://github.com/access-company/antikythera/pull/168/commits/acb0d91d980a349440fa37c5ea90303f95e3196c